README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# HW 01
+
+### Introduction
+
+![-](https://user-images.githubusercontent.com/67297411/85501690-7d090680-b5ab-11ea-9c45-1364676d9eda.png)
+
+Hello! My name is Pranathi Posa, and I'm a rising third-year in The College. I'm majoring in Public Policy and minoring in Statistics. I write and edit for *The Maroon* and *Polytics*, and I'm also a part of ChoMUN.
+
+### Here are Some of My Favorite...
+
+**Books?** *Good Omens* by Neil Gaiman, *The Martian* by Andy Weir, *Extremely Loud & Incredibly Close* by Jonathan Safran Foer
```

Comment:
Full disclosure: I cried when I watched the movie version of Extremely Loud & Incredibly Close